src/app/api/services/dot-router/dot-router.service.ts

```diff
@@ -90,6 +90,16 @@ export class DotRouterService {
         return url === '/';
     }
 
+    /**
+     * Check if Portlet is based on a JSP/iframe page
+     * @returns {boolean}
+     * @memberof DotRouterService
+     */
+    isJSPPortlet(): boolean {
+        debugger;
+        return this.router.url.startsWith('/c/');
```

Comment:
I think this constant should be returned by the backend such as:
'legacy-urls': ['/c/']

But not a big deal